src/main/java/de/tum/in/test/api/util/ReflectionTestUtils.java

```diff
@@ -287,7 +538,7 @@ public static Object invokeMethodRethrowing(Object object, Method method, Object
 	 */
 	public static <T> Constructor<T> getConstructor(Class<T> declaringClass, Class<?>... parameterTypes) {
 		try {
-			return declaringClass.getConstructor(parameterTypes);
+			return declaringClass.getDeclaredConstructor(parameterTypes);
```

Comment:
This should probably use ClassMemberAccessor as well (and have two variants)

Reply:
I reverted the change to getConstructor, as it is not used by the methods accessing the private constructors anyway.